README.md

```diff
@@ -91,7 +107,10 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+function doSomething (thingToDo) {
+  return thingToDo;
+}
+doSomething("Do This")
```

Comment:
or pass a function in: https://github.com/ga-wdi-quizzes/week-04/blame/6111775b35480b3fd9021976459e0a4402981ca0/README.md#L108-L113